docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -35,112 +32,122 @@ documented on the identity providers website. Here are a few links:
 - [Keycloak Client Registration](https://www.keycloak.org/docs/latest/securing_apps/index.html#\_client_registration)
 
-Add your OIDC connector information to `teleport.yaml`. A few examples are
-provided below.
-
 <Admonition type="tip">
-  For Google Workspace please follow our dedicated [Guide](google-workspace.mdx)
+For Google Workspace please follow our dedicated [Guide](google-workspace.mdx)
 </Admonition>
 
+Save the relevant information from your identity provider. To make following
+this guide easier, you can add the values here and they will be included in the
+example commands below:
+
+- Client ID: <Var name="<CLIENT-ID>" description="The Client ID for your Teleport application, provided by your IdP"/>
+- Client secret: <Var name="<CLIENT-SECRET>" description="The Client secret for your Teleport application, provided by your IdP"/>
+
 ## OIDC Redirect URL
 
-OIDC relies on HTTP re-directs to return control back to Teleport after
+OIDC relies on HTTP redirects to return control back to Teleport after
 authentication is complete. The redirect URL must be selected by a Teleport
 administrator in advance.
 
-If the Teleport web proxy is running on `proxy.example.com` host, the redirect URL
-should be `https://proxy.example.com:3080/v1/webapi/oidc/callback`
+The redirect URL for OIDC authentication in Teleport is <nobr><Var
+name="mytenant.teleport.sh" description="Your Teleport Cloud tenant or Proxy
+Service address"/>`/v1/webapi/oidc/callback`</nobr>. Replace `mytenant.teleport.sh`
+with your Teleport Cloud tenant or Proxy Service address.
 
 ## OIDC connector configuration
 
-The next step is to add an OIDC connector to Teleport. The connectors are manipulated
-via `tctl` [resource commands](../../reference/resources.mdx). To create a new connector,
-create a connector resource file in YAML format, for example `oidc-connector.yaml`.
+The next step is to add an OIDC connector to Teleport. The connectors are
+created, test, and added / removed via `tctl` [resource commands](../../reference/resources.mdx).
 
-The file contents are shown below. This connector requests the scope `group`
-from the identity provider then mapping the value to either to `admin` role or
-the `user` role depending on the value returned for `group` within the claims.
-
-```yaml
-(!examples/resources/oidc-connector.yaml!)
-```
-
-Create the connector:
-
-<ScopedBlock scope={["oss", "enterprise"]}>
+To create a new connector, use `tctl sso configure`. Our example creates a
+connector resource file in YAML format named `oidc-connector.yaml`:
 
 ```code
-# Log in to your cluster with tsh so you can use tctl from your local machine.
-# You can also run tctl on your Auth Service host without running "tsh login"
-# first.
-$ tsh login --proxy=teleport.example.com --user=myuser
-$ tctl create oidc-connector.yaml
+$ tctl sso configure oidc --name <CONNECTOR-NAME> \
+  --issuer-url <PATH-TO-PROVIDER> \
+  --id <CLIENT-ID> --secret <CLIENT-SECRET> \
```

Comment:
This has the side effect of leaving the client secret in the user's shell history, which isn't great. I wonder if we should suggest an alternative.

Reply:
Is there an alternative that still creates a complete connector resource?

How about a warning to consider scrubbing this command from shell history? The `--secret` key is used in several other pages where `tctl sso` is used, so we could make it a partial file.

Reply:
I don't think so, but we could have the user put the secret in a file and use `--secret $(cat client-secret.txt` or something.

Reply:
Since the use of `--secret` is now in several docs, I'll make a new issue that once this PR (#29087) is merged, we should update all the SSO guides to use this method.

> I don't think so, but we could have the user put the secret in a file and use `--secret $(cat client-secret.txt)` or something.